yarn-project/end-to-end/src/sample-dapp/index.test.mjs

```diff
@@ -16,7 +18,7 @@ describe('token', () => {
     recipient = await createAccount(pxe);
 
     const initialBalance = 69;
-    await deployToken(owner, initialBalance, createDebugLogger('sample_dapp'));
+    token = await deployToken(owner, initialBalance, createDebugLogger('sample_dapp'));
```

Comment:
This is the fix. Forgot to assign to the variable. Not having this checked by the build process does not help.